modules/ossm-installing-multi-primary-network-mesh.adoc

```diff
@@ -0,0 +1,303 @@
+----
+
+. Deploy the sample applications on the West cluster:
+
+.. Create a sample application namespace on the West cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get ns sample --context "${CTX_CLUSTER2}" || oc create --context="${CTX_CLUSTER2}" namespace sample
+----
+
+.. Label the application namespace to support sidecar injection by running the following command:
++
+[source,terminal]
+----
+$ oc label --context="${CTX_CLUSTER2}" namespace sample istio-injection=enabled
+----
+
+.. Deploy the `helloworld` application:
+
+... Create the `helloworld` service by running the following command:
++
+[source,terminal]
+----
+oc --context="${CTX_CLUSTER2}" apply \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
+  -l service=helloworld -n sample
+----
+
+... Create the `helloworld-v2` deployment by running the following command:
++
+[source,terminal]
+----
+oc --context="${CTX_CLUSTER2}" apply \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
+  -l version=v2 -n sample
+----
+
+.. Deploy the `sleep` application by running the following command:
++
+[source,terminal]
+----
+oc --context="${CTX_CLUSTER2}" apply \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/sleep/sleep.yaml -n sample
+----
+
+. Wait for the sample applications on the West cluster to return the "Ready" status condition by running the following command:
++
+[source,terminal]
+----
+$ oc --context="${CTX_CLUSTER2}" wait --for condition=available -n sample deployment/helloworld-v2
+$ oc --context="${CTX_CLUSTER2}" wait --for condition=available -n sample deployment/sleep
+----
+
+.Verification
+
+. For the East cluster, send 10 requests to the `helloworld` service by running the following command:
++
+[source,terminal]
+----
+for i in {0..9}; do
+  kubectl exec --context="${CTX_CLUSTER1}" -n sample deploy/sleep -c sleep -- curl -sS helloworld.sample:5000/hello;
+done
+----
++
+Verify that you see responses from both clusters. This means you will see version 1 and version 2 of the service in the responses. 
+
+. For the West cluster, send 10 requests to the `helloworld` service:
++
+[source,terminal]
+----
+for i in {0..9}; do
+  kubectl exec --context="${CTX_CLUSTER2}" -n sample deploy/sleep -c sleep -- curl -sS helloworld.sample:5000/hello;
+done
+----
++
+Verify that you see responses from both clusters. This means you will see version 1 and version 2 of the service in the responses. 
+
+////
+
+.Cleanup
+
+. Optional: Remove the cluster1 from a development environment by running the following command:
++
+[source,terminal]
+----
+oc --context="${CTX_CLUSTER1}" delete istio/default ns/istio-system ns/sample ns/istio-cni
+----
+
+. 
+. Optional: Remove the cluster2 from a development environment by running the following command:
++
```

Comment:
This renders incorrectly: 
![image](https://github.com/user-attachments/assets/f34d8484-c617-4e2d-94ca-4b043876e26a)

Reply:
It's fixed in the standalone topic.